pkg/image/image.go

```diff
@@ -105,6 +105,24 @@ func (img *ContainerImage) GetFullNameWithTag() string {
 	return str
 }
 
+// GetFullTag returns tag name along with any tag digest set for the image
+func (img *ContainerImage) GetFullTag() string {
```

Comment:
Can we please rename this to something more precise, e.g. `GetTagWithDigest()`?

Reply:
Will do immediately!

Reply:
Done.